src/ui/reducers/app.ts

```diff
@@ -252,8 +252,37 @@ export const isInspectorSelected = (state: UIState) =>
   getViewMode(state) === "dev" && getSelectedPanel(state) == "inspector";
 export const getInitializedPanels = (state: UIState) => state.app.initializedPanels;
 export const getRecordingDuration = (state: UIState) => state.app.recordingDuration;
-export const getIndexing = (state: UIState) => state.app.indexing;
-export const getIndexed = (state: UIState) => state.app.indexing == 100;
+export const getIndexingProgress = (state: UIState) => {
+  const regions = getLoadedRegions(state);
+
+  if (!regions) {
+    return null;
+  }
+
+  const { loading, indexed } = regions;
+
+  const indexedProgress = indexed
```

Comment:
:hand: This looks like a good candidate for a memoized Reselect `createSelector` version rather than a plain function.  Not sure how big `indexed` or `loading` are, but a `.some()` inside a `.filter()` looks like it's worth avoiding if we don't need to recalculate it.